frontend/package.json

```diff
@@ -26,6 +26,7 @@
     "@types/recharts": "^1.8.13",
     "@types/uuid": "^8.0.0",
     "dayjs": "^1.8.36",
+    "immer": "8.0.1",
```

Comment:
We don't need a direct dependency on this library so is not a good idea to add it to our package.json. Instead, we need to see what library we depend on uses it, and see if that library has already released a patch.

Reply:
react-dev-utils is being upgraded to immer 8.0.1.  Let's track this [issue](https://github.com/facebook/create-react-app/issues/10411)

Reply:
Removed from package.json. But I guess we can wait and see if they fix it in react dev utils.